packages/console/src/assets/images/logo.svg

```diff
@@ -0,0 +1 @@
1 21.096h-15.541c-14.06 0-21.091-7.032-21.091-21.096V99.096c0-14.064 7.031-21.096 21.091-21.096h15.541c14.06 0 21.091 7.032 21.091 21.096v59.958Z" fill="currentColor"/><path d="M0 37.512c0-8.837 7.163-16 16-16h170.166c8.837 0 16 7.163 16 16v183.126c0 8.836-7.163 16-16 16H16c-8.837 0-16-7.164-16-16V37.512Z" fill="url(#paint0_linear_935_668)"/><path d="M231.201 273.748 34.949 235.025l283.356 3.227-87.104 35.496Z" fill="url(#paint1_linear_935_668)"/><path d="M0 35.389a16 16 0 0 1 13.737-15.84L132.286 2.61c9.639-1.377 18.263 6.103 18.263 15.84v221.253c0 9.737-8.624 17.217-18.263 15.84l-118.55-16.94A16 16 0 0 1 0 222.762V35.389Z" fill="url(#paint2_linear_935_668)"/><g filter="url(#filter0_d_935_668)"><ellipse cx="116.138" cy="129.076" rx="12.904" ry="12.908" fill="url(#paint3_linear_935_668)"/></g><g filter="url(#filter1_d_935_668)"><rect x="109.686" y="123.697" width="12.904" height="34.42" rx="6.452" fill="url(#paint4_linear_935_668)"/></g><defs><linearGradient id="paint0_linear_935_668" x1="4.839" y1="136.604" x2="205.001" y2="72.586" gradientUnits="userSpaceOnUse"><stop/><stop offset="1"/></linearGradient><linearGradient id="paint1_linear_935_668" x1="118.827" y1="241.479" x2="240.344" y2="266.206" gradientUnits="userSpaceOnUse"><stop/><stop offset="1"/></linearGradient><linearGradient id="paint2_linear_935_668" x1="-12.367" y1="147.899" x2="162.382" y2="110.271" gradientUnits="userSpaceOnUse"><stop/><stop offset="1"/></linearGradient><linearGradient id="paint3_linear_935_668" x1="116.138" y1="116.168" x2="110.759" y2="141.983" gradientUnits="userSpaceOnUse"><stop/><stop offset="1"/></linearGradient><linearGradient id="paint4_linear_935_668" x1="116.138" y1="123.697" x2="100.89" y2="151.138" gradientUnits="userSpaceOnUse"><stop/><stop offset="1"/></linearGradient><filter id="filter0_d_935_668" x="102.234" y="116.168" width="27.808" height="27.814" filterUnits="userSpaceOnUse"><feFlood result="BackgroundImageFix"/><feColorMatrix in="SourceAlpha" values="0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 127 0" result="hardAlpha"/><feOffset dy="1"/><feGaussianBlur stdDeviation=".5"/><feComposite in2="hardAlpha" operator="out"/><feColorMatrix values="0 0 0 0 0.104304 0 0 0 0 0.0825 0 0 0 0 0.2 0 0 0 0.02 0"/><feBlend in2="BackgroundImageFix" result="effect1_dropShadow_935_668"/><feBlend in="SourceGraphic" in2="effect1_dropShadow_935_668" result="shape"/></filter><filter id="filter1_d_935_668" x="105.686" y="123.697" width="18.904" height="41.42" filterUnits="userSpaceOnUse"><feFlood result="BackgroundImageFix"/><feColorMatrix in="SourceAlpha" values="0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 127 0" result="hardAlpha"/><feOffset dx="-1" dy="4"/><feGaussianBlur stdDeviation="1.5"/><feComposite in2="hardAlpha" operator="out"/><feColorMatrix values="0 0 0 0 0.104304 0 0 0 0 0.0825 0 0 0 0 0.2 0 0 0 0.08 0"/><feBlend in2="BackgroundImageFix" result="effect1_dropShadow_935_668"/><feBlend in="SourceGraphic" in2="effect1_dropShadow_935_668" result="shape"/></filter></defs></svg>
```

Comment:
I found multiple things are missing in the `<defs></defs`>, and it seems it's not directly copied from the original `Logo.svg`. What tool did you use?

Reply:
`pnpx svgo logo.svg`

But maybe I also messed something up when copying from tsx. I'll have a look.

Reply:
ok, thanks!
Ah, I just realized it could be caused by the `camelCaseAttributes` that used in the tsx that are not recognized by `pnpx`. So the tool trim them all off and therefore ended up like this.

You'll probably have to manually add those attributes back. Something like `floodOpacity`, `colorInterpolationFilters` and `stopColor`, which should've been in `kebab-case` in HTML/SVG standard.